duffy/app/main.py

```diff
@@ -2,10 +2,29 @@
 
 from fastapi import FastAPI
 
+from ..version import __version__
 from .controllers import chassis, node, session, tenant
 
 log = logging.getLogger(__name__)
-app = FastAPI()
+
+description = """
+Duffy is the middle layer running [`ci.centos.org`](https://ci.centos.org). It provisions, tears
+down and rebuilds physical and virtual machines are used to run tests in the CentOS CI Cluster.
+"""
+
+tags_metadata = [
+    {"name": "nodes", "description": "Operations with physical and virtual nodes"},
+    {"name": "sessions", "description": "Operations with sessions"},
+    {"name": "tenants", "description": "Operations with tenants"},
```

Comment:
Just noticed that this is missing an entry for `chassis`, would you add it?

Reply:
Done in https://github.com/CentOS/duffy/pull/133/commits/be9652cb91922a8ac5451a81a57be54eae6dee47. Good for another look.